### PR TITLE
Remove unnecessary extra lookup tables in tests

### PR DESCRIPTION
**Pull request by mrmr1993**

### CODE REVIEW

File: kimchi/src/tests/range_check.rs [hunk 1]
```diff
@@ -645,7 +644,6 @@ fn verify_range_check0_v1_test_lookups() {
             TestFramework::<Vesta>::default()
                 .gates(index.cs.gates.clone())
                 .witness(witness.clone())
-                .lookup_tables(vec![range_check::gadget::lookup_table()])
```
Comment (jspada) [inline kimchi/src/tests/range_check.rs:648]: Oh, interesting. Are the lookup constraints still working without this?  Sounds like I've misunderstood something!

So it automatically knows to include the range-check lookup table without needing to specify it here?  And, by doing it here I was including an extra copy?

**Review by jspada — APPROVED** [review]

File: kimchi/src/tests/range_check.rs (hunk 1 above)
Comment (mrmr1993) [reply kimchi/src/tests/range_check.rs:648]: That's right, yea. The constraint system generator already associates gates with particular lookup tables

Comment (jspada) [reply kimchi/src/tests/range_check.rs:648]: That's great.  Is it something new or has it always done that?

Comment (mrmr1993) [reply kimchi/src/tests/range_check.rs:648]: It's always done that!

**Pull request merged**